seatunnel-e2e/seatunnel-flink-connector-v2-e2e/connector-flink-e2e-base/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
move this file to `connector-doris-flink-e2e` module?

reference
https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-flink-connector-v2-e2e/connector-jdbc-flink-e2e/src/test/resources